packages/cactus-plugin-ledger-connector-iroha/src/main/typescript/plugin-ledger-connector-iroha.ts

```diff
@@ -0,0 +1,684 @@
+    throw new Error("Method not implemented.");
+  }
+
+  public getPrometheusExporter(): PrometheusExporter {
+    return this.prometheusExporter;
+  }
+
+  public async getPrometheusExporterMetrics(): Promise<string> {
+    const res: string = await this.prometheusExporter.getPrometheusMetrics();
+    this.log.debug(`getPrometheusExporterMetrics() response: %o`, res);
+    return res;
+  }
+
+  public getInstanceId(): string {
+    return this.instanceId;
+  }
+
+  public async onPluginInit(): Promise<unknown> {
+    return;
+  }
+
+  public getHttpServer(): Optional<Server | SecureServer> {
+    return Optional.ofNullable(this.httpServer);
+  }
+
+  public async shutdown(): Promise<void> {
+    const serverMaybe = this.getHttpServer();
+    if (serverMaybe.isPresent()) {
+      const server = serverMaybe.get();
+      await promisify(server.close.bind(server))();
+    }
+  }
+
+  async registerWebServices(
+    app: Express,
+    //wsApi: SocketIoServer,
+  ): Promise<IWebServiceEndpoint[]> {
+    //const { web3 } = this;
+    //const { logLevel } = this.options;
+    const webServices = await this.getOrCreateWebServices();
+    await Promise.all(webServices.map((ws) => ws.registerExpress(app)));
+
+    // wsApi.on("connection", (socket: SocketIoSocket) => {
+    //   this.log.debug(`New Socket connected. ID=${socket.id}`);
+
+    //   socket.on(WatchBlocksV1.Subscribe, () => {
+    //     new WatchBlocksV1Endpoint({ web3, socket, logLevel }).subscribe();
+    //   });
+    // });
+    return webServices;
+  }
+
+  public async getOrCreateWebServices(): Promise<IWebServiceEndpoint[]> {
+    if (Array.isArray(this.endpoints)) {
+      return this.endpoints;
+    }
+    const endpoints: IWebServiceEndpoint[] = [];
+    {
+      const endpoint = new RunTransactionEndpoint({
+        connector: this,
+        logLevel: this.options.logLevel,
+      });
+      endpoints.push(endpoint);
+    }
+    // {
+    //   const endpoint = new InvokeContractEndpoint({
+    //     connector: this,
+    //     logLevel: this.options.logLevel,
+    //   });
+    //   endpoints.push(endpoint);
+    // }
+
+    {
+      const opts: IGetPrometheusExporterMetricsEndpointV1Options = {
+        connector: this,
+        logLevel: this.options.logLevel,
+      };
+      const endpoint = new GetPrometheusExporterMetricsEndpointV1(opts);
+      endpoints.push(endpoint);
+    }
+    this.endpoints = endpoints;
+    return endpoints;
+  }
+
+  public getPackageName(): string {
+    return `@hyperledger/cactus-plugin-ledger-connector-iroha`;
+  }
+
+  public async getConsensusAlgorithmFamily(): Promise<
+    ConsensusAlgorithmFamily
+  > {
+    return ConsensusAlgorithmFamily.Authority;
+  }
+  public async hasTransactionFinality(): Promise<boolean> {
+    const currentConsensusAlgorithmFamily = await this.getConsensusAlgorithmFamily();
+
+    return consensusHasTransactionFinality(currentConsensusAlgorithmFamily);
+  }
+  //...  spread
```

Comment:
@hxlaf Please delete the commented lines unless there's a specific reason to keep them around.

Reply:
Removed! Thank you!